src/main/java/Ui.java

```diff
@@ -0,0 +1,130 @@
+     this method is only used in Level-1 of iP
+    */
+    public void echo(String s) {
+        System.out.println(s);
+    }
+
+    // this method prompts the user for a command
+    public String readCommand() {
+        Scanner sc = new Scanner(System.in);
+        System.out.println("     Give me a command! ");
+        return sc.nextLine();
+    }
+
+    public void showLine() {
+        System.out.println(LINE);
+    }
+
+    public void unknownCommand(String command) {
+        System.out.println("     ☹ OOPS!!! I'm sorry, but I don't know what \"" + command + "\" means :-(");
+    }
+
+    public void createToDoSuccessMessage(ToDo t, int numOfTasks) {
+        System.out.println("     Got it. I've added this task: ");
+        System.out.println("     " + t.toString());
+        System.out.println("     Now you have " + numOfTasks + " tasks in the list.");
+    }
+
+    public void printTasks(ArrayList<Task> tasks) {
+        if (tasks.size() == 0) {
+            System.out.println("     Currently, you have no tasks!");
+        } else {
+            System.out.println("     These are your tasks:");
+            int counter = 1;
+            for (Task task : tasks) {
+                if (task instanceof Deadline) {
+                    System.out.println("     " + counter + ". " + ((Deadline) task).toString());
+                    counter++;
+                } else if (task instanceof Event) {
+                    System.out.println("     " + counter + ". " + ((Event) task).toString());
+                    counter++;
+                } else if (task instanceof ToDo) {
+                    System.out.println("     " + counter + ". " + ((ToDo) task).toString());
+                    counter++;
+                }
+            }
+        }
+    }
+
+    public void doneCommandSuccessMessage(int taskNumber, Task t) {
+        System.out.println("     Nice! I've marked this task as done: ");
+
+        if (t instanceof Event) {
+            System.out.println("       " + taskNumber + ". "+ ((Event)t).toString());
+        } else if (t instanceof Deadline) {
+            System.out.println("       " + taskNumber + ". "+ ((Deadline)t).toString());
+        } else if (t instanceof ToDo) {
+            System.out.println("       " + taskNumber + ". "+ ((ToDo)t).toString());
+        }
+    }
+
+    public void createDeadlineSuccessMessage(Deadline d, int numOfTasks) {
+        System.out.println("     Got it. I've added this task: ");
+        System.out.println("     " + d.toString());
+        System.out.println("     Now you have " + numOfTasks + " tasks in the list.");
+    }
+
+    public void createEventSuccessMessage(Event e, int numOfTasks) {
+        System.out.println("     Got it. I've added this task: ");
+        System.out.println("     " + e.toString());
+        System.out.println("     Now you have " + numOfTasks + " tasks in the list.");
+    }
+
+    public void showError(String message) {
```

Comment:
I think it will be better if we standardise this to `showErrorMessage` as per the other methods?